src/BenchmarkDotNet/Running/BuildPartition.cs

```diff
@@ -60,5 +62,11 @@ public BuildPartition(BenchmarkBuildInfo[] benchmarks, IResolver resolver)
         public bool IsCustomBuildConfiguration => BuildConfiguration != InfrastructureMode.ReleaseConfigurationName;
 
         public override string ToString() => RepresentativeBenchmarkCase.Job.DisplayInfo;
+
+        private static string GetResolvedAssemblyLocation(Assembly assembly) =>
+            // in case of SingleFile, location.Length returns 0, so we use GetName() and
+            // manually construct the path.
+            assembly.Location.Length == 0 ?
+                Path.Combine(AppContext.BaseDirectory, assembly.GetName().Name) : assembly.Location;
```

Comment:
just to trigger the CI

```suggestion
            assembly.Location.Length == 0 ? Path.Combine(AppContext.BaseDirectory, assembly.GetName().Name) : assembly.Location;
```